.github/workflows/ci.yaml

```
@@ -874,7 +874,7 @@ jobs:
     if: |
       (github.event_name != 'push' || github.event.repository.full_name == 'home-assistant/core')
       && github.event.inputs.lint-only != 'true'
-      && (needs.info.outputs.test_full_suite == 'true' || needs.info.outputs.tests_glob)
+      && (needs.info.outputs.test_full_suite == 'true' || contains(needs.info.outputs.tests_glob, 'recorder'))
```

There is also the recorder platform, I think it is better to use the list of integrations that have changed.

```suggestion
      && (needs.info.outputs.test_full_suite == 'true' || contains(needs.info.outputs.integrations, 'recorder'))
```

---

Ooh, it is even simpler:

<https://github.com/home-assistant/core/blob/dev/.core_files.yaml#L89>

So, should we instead only run this job when a full suite is requested?

---

Saw your second comment too late.

> So, should we instead only run this job when a full suite is requested?

AFAICT any `contains` was only a nice to have. `needs.info.outputs.test_full_suite == 'true'` should always trigger anyway. So yes, that will be enough.